confirmDialog.js

```diff
@@ -0,0 +1,65 @@
+const St = imports.gi.St;
+const GObject = imports.gi.GObject;
+const ModalDialog = imports.ui.modalDialog;
+const CheckBox = imports.ui.checkBox;
+const Clutter = imports.gi.Clutter;
+
+const Gettext = imports.gettext.domain("gnome-trash");
```

Comment:
You forgot to edit this ? We should use extension context

Reply:
Good find. Copy-pasting issue. Thanks man!